syntax = "proto3";

package protodb;

message ListDatabasesRequest {
}

message ListDatabasesResponse {
    repeated string database = 1;
}